docs/instant.md

```diff
@@ -178,52 +178,46 @@ sorted.join(' ');
 // => 2001-09-09T01:46:40Z 2004-11-09T11:33:20Z 2008-01-10T21:20Z
 ```
 
-## Methods
-
-### instant.**getEpochSeconds**() : number
+## Properties
 
-**Returns:** an integer number of seconds.
+### instant.**epochSeconds** : number
 
-Returns the number of seconds between the Unix epoch (midnight UTC on January 1, 1970) and `instant`.
+The value of this property is an integer number of seconds between the Unix epoch (midnight UTC on January 1, 1970) and `instant`.
 This number will be negative if `instant` is before 1970.
 The number of seconds is rounded towards zero.
 
-Use this method if you need to interface with some other system that reckons time in seconds since the Unix epoch.
+Use this property if you need to interface with some other system that reckons time in seconds since the Unix epoch.
 
 Example usage:
 
 ```js
 instant = Temporal.Instant.from('2019-03-30T01:45+01:00');
-instant.getEpochSeconds(); // => 1554000300
+instant.epochSeconds; // => 1554000300
 ```
 
-### instant.**getEpochMilliseconds**() : number
-
-**Returns:** an integer number of milliseconds.
+### instant.**epochMilliseconds** : number
 
-Same as `getEpochSeconds()`, but with millisecond (10<sup>&minus;3</sup> second) precision.
+Same as `epochSeconds`, but with millisecond (10<sup>&minus;3</sup> second) precision.
```

Comment:
I think it'd be helpful to repeat the content about rounding towards zero here and in the property below. This is implied from "same as `epochSeconds`" but the rounding behavior is an important detail to understand so it's worth repeating. 

Also, minor nit: shouldn't we call it "truncated towards zero" instead of "rounded towards zero" to be more clear about what it does?  To me "rounding" implies that the larger unit might sometimes change while "truncation" forecloses that possibility.